docs/_documentations/codewind-minikube-guickguide.md

```diff
@@ -0,0 +1,110 @@
+
+* [Prerequisistes](##prerequisites)
+* [Preparing Minikube](##preparing-minikube)
+* [Installing Codewind](##installing-codewind)
+* [Developing a microservice](##developing-a-microservice)
+* [What you have learned](##what-you-have-learned)
+* [Next steps](##next-steps) 
+
+## Prerequisites
+* A Mac, Windows, or Linux host.
+* Install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+* At least 8Gb RAM. Minikube needs at least 4Gb of RAM for this tutorial. 
+
+## Preparing Minikube
+1. To run the needed commands, open your terminal. 
+2. To start Minikube on your machine, run, `minikube start --memory=4096`.
+    * Minikube creates a single-node Kubernetes cluster with 4GB of RAM available to it.
+    * If this command fails, refer to [Minikube installation docs](https://kubernetes.io/docs/tasks/tools/install-minikube) and ensure you have enough memory on your system. 
+3. To install [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) on Minikube, run, `minikube addons enable ingress`. 
+    * Codewind makes use of Ingress to expose its endpoints.
+4. To determine your Ingress IP, run `minikube ip`. 
+    * Example:
+     ```
+     johns-mbp-3:~ johncollier$ minikube ip
+     192.168.64.46
+     ```
+     * Save this value for later.
+ 
+## Installing Codewind
+1. Choose your IDE and install the Codewind extension. 
+    * Follow the install instructions for [Eclipse](https://www.eclipse.org/codewind/mdt-eclipse-getting-started.html) or [VS Code](https://www.eclipse.org/codewind/mdt-vsc-getting-started.html). 
+2. Locate the `cwctl` command line interface (CLI). 
+    * On Mac and Linux, the cwctl is `~/.codewind/<version>/cwctl`. 
+    * On Windows, the cwctl is located: ??? 
+3. To install cwctl, run the following command:
+        ```
+        ./cwctl --insecure install remote \
+        --namespace codewind \
+        --kadminuser admin \
+        --kadminpass admin \
+        --krealm codewind \
+        --kclient codewind \
+        --kdevuser developer \
+        --kdevpass developer \
+        --ingress <ingress-ip>.nip.io
+        ```   
+    * **Note:** The `ingress-ip` value is the value you found in Step 4 of `Preparing Minikube`.
+    * This command installs Codewind into the `Codewind` namespace with the default credentials.
+4. Open the Codewind view in your editor -> select `New Codewind Connection` -> enter the following details:   
+    * **Codewind Gatekeeper URL:** In step 3, cwctl gives you the Gatekeeper URL. 
+    * **Username:** In step 3, developer username is given to cwctl in the value of `kdevuser`. 
+    * **Password:** In step 3, developer password given to cwctl in the value of `kdevpass`.
+5. To save and create the connection to the Codewind instance on Minikube, click `Save`. 
+
+## Developing a microservice
```

Comment:
We can re-use the developing a microservice section from the local guide here

Reply:
Are you talking about the [Getting started with Codewind in VS Code guide](https://kabanero.io/guide/codewind-getting-started-vscode/) and [Getting started with Codewind in Eclipse guide](https://kabanero.io/guide/codewind-getting-started-eclipse/)? 

The instructions for developing a microserice differ between VS Code and Eclipse, so they won't be the same, pending on the IDE users work in. Maybe we could link to them, that would defeat the purpose of a quick, self-contained guide though. 

In my new commit, I've added the instructions for developing a microservice within VS Code. Maybe we can tailor it more to Minikube? Right now, it's solely developing a microservice, using Codewind on VS Code.